commit f25efbef5080f1223bb0e081dbb9c36decae4e7a (rdkcentral/feature/test_push, rdkcentral/feature/RDKEVD-3467DisableZRAM)
Author: Aravindan NC <35158113+AravindanNC@users.noreply.github.com>
Date:   Thu Oct 2 15:18:24 2025 -0400

 RDKE-921: Clean up the mitigation done for dnsmasq restart in Xumo TV Release (#217)

Date: Tue, 16 Jun 2026 12:59:41 +0000
From: Franciszek Koltuniuk <franciszek.2173280@infosys.com>

 RDKDEV-1326: Upgrade NetworkManager to v1.57.4

---
 src/core/dns/nm-dns-dnsmasq.c         | 2 +-
 src/core/dnsmasq/nm-dnsmasq-manager.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/core/dns/nm-dns-dnsmasq.c b/src/core/dns/nm-dns-dnsmasq.c
index cc0a878843..8ec8c394a7 100644
--- a/src/core/dns/nm-dns-dnsmasq.c
+++ b/src/core/dns/nm-dns-dnsmasq.c
@@ -521,7 +521,7 @@ _gl_pid_spawn_next_step(void)
     argv[argv_idx++] = "--no-resolv"; /* Use only commandline */
     argv[argv_idx++] = "--keep-in-foreground";
     argv[argv_idx++] = "--no-hosts"; /* don't use /etc/hosts to resolve */
-    argv[argv_idx++] = "--bind-dynamic";
+    // argv[argv_idx++] = "--bind-dynamic";
     argv[argv_idx++] = "--pid-file=" PIDFILE;
     argv[argv_idx++] = "--listen-address=127.0.0.1";
     argv[argv_idx++] = "--listen-address=::1";
diff --git a/src/core/dnsmasq/nm-dnsmasq-manager.c b/src/core/dnsmasq/nm-dnsmasq-manager.c
index 030a8e28e1..0eb3e7f09e 100644
--- a/src/core/dnsmasq/nm-dnsmasq-manager.c
+++ b/src/core/dnsmasq/nm-dnsmasq-manager.c
@@ -141,7 +141,7 @@ create_dm_cmd_line(const char           *iface,
 
     nm_strv_ptrarray_add_string_dup(cmd, "--no-hosts");
     nm_strv_ptrarray_add_string_dup(cmd, "--keep-in-foreground");
-    nm_strv_ptrarray_add_string_dup(cmd, "--bind-interfaces");
+    // nm_strv_ptrarray_add_string_dup(cmd, "--bind-interfaces");
     nm_strv_ptrarray_add_string_dup(cmd, "--except-interface=lo");
     nm_strv_ptrarray_add_string_dup(cmd, "--clear-on-reload");
 
-- 
2.30.0
